### ISSUE TITLE
Review alignment and sizes of icons

### ISSUE THREAD

**Issue by hunvreus** [Wiredcraft/twig#6]
I was pointed at the fact that currently, the twig icons aren't really lining up properly with a grid. For example:

![screenshot at oct 07 23-04-07](https://cloud.githubusercontent.com/assets/306868/10356706/f46b2084-6d47-11e5-8fec-135bd1813fe9.png)

While it's fine to have curves or occasional elements that don't fit in the grid, we should try and align as many of the straight lines and outlines to the 16 or 24 grid (depending on the icon set) for readability and ensuring the edges of the icons are sharp.

You can ask me if you need more details, but the picture is pretty self descriptive.

**Comment by Iriszhousy** [comment]
So my job is to redesign all this icons:make sure the icon can fit in the grid. So what I should do is to move the shapes to the edges? 
And where I can find the resource.

**Comment by quentinberder** [comment]
It should be in the Google Drive, @hunvreus where could we find them? I see only the genrated svg in the repository.

**Comment by Iriszhousy** [comment]
There is nothing in my Google Drive.

**Comment by quentinberder** [comment]
Will sync tomorrow (VPN issues I presume).

**Comment by Iriszhousy** [comment]
@hunvreus  Jan helped me find some reference. So I tried 3 icons first. I wasted time about misunderstand what you said, sorry about that. But I think this time is better, and I'll finish it as soon as possible.
![screen shot 2015-10-19 at 5 26 03 pm](https://cloud.githubusercontent.com/assets/14867627/10574316/cef7d830-7687-11e5-97c1-2e828a6834d5.png)
![screen shot 2015-10-19 at 5 26 14 pm](https://cloud.githubusercontent.com/assets/14867627/10574319/d6316314-7687-11e5-92f8-0b380250c281.png)
![screen shot 2015-10-19 at 5 26 24 pm](https://cloud.githubusercontent.com/assets/14867627/10574321/de6fda24-7687-11e5-916e-44fd95131efa.png)
